app/domain/search/audit_filter.rb

```diff
@@ -0,0 +1,42 @@
+class Search
+  class AuditFilter
+    AUDIT_STATUSES_IDENTIFIERS = [
+      :audited,
+      :non_audited,
+    ].freeze
+
+    attr_accessor :status
+
+
+    def initialize(status)
+      self.status = status ? status.to_sym : nil
```

Comment:
Good point!, let's pair so we decide where to put the validation: in the controller or in Search.